pioneer/packages/joy-proposals/src/forms/GenericWorkingGroupProposalForm.tsx

```diff
@@ -45,7 +45,17 @@ export type FormInnerProps = ProposalFormInnerProps<FormContainerProps, FormValu
 
 const availableGroupsOptions = Object.keys(WorkingGroupDef)
   .filter((wgKey) => wgKey !== 'Gateway') // Gateway group not yet supported!
-  .map((wgKey) => ({ text: wgKey + ' Working Group', value: wgKey }));
+  .map((wgKey) => {
+    let text = wgKey + 'Working Group';
```

Comment:
I think you missed the spacebar:
```suggestion
    let text = `${wgKey} Working Group`;
```